src/test/java/domain/LottosTest.java

```diff
@@ -30,13 +28,35 @@ public void getWinningStatisticsTest() {
             List.of(LottoNumber.valueOf(1), LottoNumber.valueOf(2), LottoNumber.valueOf(3),
                 LottoNumber.valueOf(4), LottoNumber.valueOf(5), LottoNumber.valueOf(6)));
         LottoNumber bonusBall = LottoNumber.valueOf(7);
-        Lottos lottos = Lottos.generateLottos(count);
-        lottos.generateLottos(count);
+        Lottos lottos = Lottos.generateLottos(new ArrayList<>(), count);
         Statistic statistic = lottos.getWinningStatistics(winningNumber, bonusBall);
 
         int sumOfValues = statistic.getStatistics().values().stream()
             .mapToInt(Integer::intValue)
             .sum();
+
         assertEquals(sumOfValues, 5);
     }
+
+    @ParameterizedTest
+    @DisplayName("생성된 로또에 대해 당첨 결과가 모두 계산되었는지 테스트")
+    @CsvSource(value = {"1,2,3,4,5,6,FIRST", "1,2,3,4,5,7,SECOND", "1,2,3,4,5,8,THIRD",
+        "1,2,3,4,7,8,FOURTH", "1,2,3,8,9,10,FIFTH", "1,2,9,10,11,12,SIXTH"})
+    public void getWinningStatisticsTestByRank(int number1, int number2, int number3, int number4, int number5, int number6, String rank) {
```

Comment:
여러가지 경우에 대해서 테스트 잘 해주셨어요 👍
이 경우는 `@MethodSource`를 사용하는게 조금 더 깔끔할 거 같다는 생각도 드네요.

Reply:
```suggestion
    @CsvSource(value = {"1,2,3,4,5,6,FIRST", "1,2,3,4,5,7,SECOND", "1,2,3,4,5,8,THIRD",
        "1,2,3,4,7,8,FOURTH", "1,2,3,8,9,10,FIFTH", "1,2,9,10,11,12,SIXTH"})
    public void getWinningStatisticsTestByRank(int number1, int number2, int number3, int number4, int number5, int number6, Rank rank) {
```
요렇게 바로 enum 타입을 선언할 수도 있습니다.